static/data/translation-table.json

```
@@ -366,6 +400,23 @@
       "en": "pickled cowpea",
       "category": "vegetable"
     },
+    {
+      "cn": "豌豆",
+      "en": "garden pea",
```

```suggestion
      "en": "pea",
```
garden pea 专指园豌豆吧？好像一般大家介绍菜谱的时候不会说得那么细
https://zh.wikipedia.org/wiki/%E8%B1%8C%E8%B1%86